### PR TITLE
[FAI-5988] Fix bootloader vulnerabilities

### PR DESCRIPTION
**Pull request by JoshSauter**
Fix vulnerabilities found in airbyte bootloader Docker image by upgrading direct and transitive dependencies.

Scan with 0 critical or high vulnerabilities: https://us-east-1.console.aws.amazon.com/ecr/repositories/private/385658942031/airbyte/_/image/sha256:774c9063ba331c2d3cb12a188515e88a0cc3f7163a60180945e340e59eef2d30/scan-results?region=us-east-1

### CODE REVIEW

File: deps.toml [hunk 1]
```diff
@@ -74,8 +74,8 @@ platform-testcontainers-postgresql = { module = "org.testcontainers:postgresql",
 log4j-over-slf4j = { module = "org.slf4j:log4j-over-slf4j", version.ref = "slf4j" }
 appender-log4j2 = { module = "com.therealvan:appender-log4j2", version = "3.6.0" }
 aws-java-sdk-s3 = { module = "com.amazonaws:aws-java-sdk-s3", version = "1.12.6" }
-google-cloud-storage = { module = "com.google.cloud:google-cloud-storage", version = "2.2.2" }
-s3 = { module = "software.amazon.awssdk:s3", version = "2.16.84" }
+google-cloud-storage = { module = "com.google.cloud:google-cloud-storage", version = "2.13.0" }
```
Comment (ypc-faros) [inline deps.toml:77]: Is it possible to instead add constraints on the transitive dependencies that these pull, same as you did above?

Comment (ypc-faros) [reply deps.toml:77]: Actually this sounds good to me...

**Comment by ypc-faros** [comment]
@cjwooo We've found a way to "force" a version of a dependency using [constraints](https://docs.gradle.org/current/userguide/dependency_constraints.html#sec:adding-constraints-transitive-deps)

Sometimes we can solve multiple issues by upgrading a dependency. Do you have a preference on either approach? Or are we okay with whatever as long as it fixes the vulns?

**Comment by cjwooo** [comment]
Up to you, if upgrading a dep version makes things easier / less things to change, then that sgtm.

**Review by cjwooo — APPROVED** [review]

**Pull request merged**